test/integration/managed_instance_group/controls/www.rb

```diff
@@ -15,11 +15,38 @@
 http_address = attribute('http_address')
 port = attribute('http_port')
 
+timeout_short_circuit = false
+
 control "www" do
   title "WWW Access"
 
-  describe http("http://#{http_address}:#{port}/", method: 'GET', open_timeout: 30, read_timeout: 30) do
-    its('status') { should eq 200 }
-    its('body') { should include 'Hello, world!' }
+  describe "HTTP service" do
+    let(:resource) do
+      raise Timeout::Error if timeout_short_circuit
+      begin
+        Timeout::timeout(600) do
```

Comment:
Given the extensive timeouts of the `resource` blocks, they should periodically print out a status message so the user is not wondering why the verification has hung.

Reply:
I don't disagree, but this isn't new code that is specifically relevant to CI. Happy to address this in a follow-on PR.

Reply:
@aaron-lane Taking a look into this, I'm not aware of a way to show output before the `inspec` run finishes. I didn't go too far down the rabbit hole, but it looks like the eventual usage of `RSpec::Core::Runner` to run the example group waits until the run completes before printing its output to the screen.